.github/workflows/e2e/k8s/collector-helm-values.yml

```diff
@@ -0,0 +1,89 @@
+mode: "statefulset"
+
+presets:
+  kubernetesAttributes:
+    enabled: true
+
+config:
+  exporters:
+    logging: {}
+    file/trace:
+      path: /tmp/trace.json
+    file/metric:
+      path: /tmp/metric.json
+  extensions:
+    health_check: {}
+    memory_ballast: {}
+  processors:
+    batch: {}
+    memory_limiter: null
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+          endpoint: ${MY_POD_IP}:4317
+    prometheus:
+      config:
+        scrape_configs:
+          - job_name: opentelemetry-collector
+            scrape_interval: 10s
+            static_configs:
+              - targets:
+                  - ${MY_POD_IP}:8888
+  service:
+    telemetry:
+      metrics:
+        address: ${MY_POD_IP}:8888
+    extensions:
+      - health_check
+      - memory_ballast
```

Comment:
All these lines are defined in the default values.yaml. Let's not duplicate them here, but keep only overrides only